src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_Misc.cs

```diff
@@ -1068,6 +1068,9 @@ public void Dev10_40561()
             return;
         }
 
+        // Test failure on ILC: Test depends on Xml Serialization and requires reflection on a LOT of types under System.Xml.Schema namespace.
+        // Rd.xml with "<Namespace Name="System.Xml.Schema" Dynamic="Required Public" />" lets this test pass but we should probably be
+        // fixing up XmlSerializer's own rd.xml rather than the test here.
```

Comment:
Nope. The fact that the test will continue failing on ILC will keep it on our radar.